applied1/survey.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Sage Bistro</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+  <link rel="stylesheet" href="_css/al1.css">
+  <link rel="stylesheet" href="themes/ctheme1.css">
+
+</head>
+
+<body>
+  <!-- Benjamin Cook
+     assignment?.html
+     INFO2439
+     Thoendel
+     9/4/2019
+-->
+  <div data-role="page" id="homePage">
+    <div data-role="header">
+      <a data-ajax="false" href="../index.html" data-icon="home" rel="external">Home</a>
+      <p style="text-align:center;"><img src="photos/sagelogo.png" alt="Sage Bistro Logo"></p>
+    </div>
+    <div class="topnav">
+        <a data-ajax="false" href="menu.html">Menu</a>
+        <a data-ajax="false" href="reservation.html">Reservations</a>
+        <a data-ajax="false" href="survey.html">Survey</a>
+        <a data-ajax="false" href="map.html">Location</a>
+    </div>
+    <form method="post" action="https://www.mccinfo.net/webresources/ogi-bin/formmail.asp">
```

Comment:
here is where I would expect to see that php script on my server. Bummer! This won't write to our db via the script you've listed.